src/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PokeCard</title>
+</head>
+<body>
+  <div class="modalBackground hidden">
+    <div class="modal">
+      <section class="cardTotal">
+        <div class="cardModal"></div>
+        <canvas id="grafica"></canvas>
+        <button class="close">X</button>
+      </section>
+      <section class="commentsTotal">
+        <div id="comments-container">
+          <div class="messagesAndCount">
+            <h2>Comments </h2>
+            <div class= "messageCount"></div>
+          </div>
+          <div class="comments"></div>
+        </div>
+        <div class="form">
+          <form action="submit">
+            <ul class="ul-form">
+              <input class="name-form"
+              type="text"
+              placeholder="Your name"
+              required>
+              <input class="insights-form"
+              type="text"
+              placeholder="Your insights"
+              required >
+              <button type="submit" class="commentBtn" id="commentBtn">Comment</button>
+          </ul>
+        </form>
+       </div>    
+      </section>
+    </div>
+  
+  </div>
+  <header>
+    <nav>
+      <img class="logo" src="https://www.freepnglogos.com/uploads/pokemon-symbol-logo-png-31.png" alt="pokemon symbol logo png" />
+        <ul class="nav-menu-ul">
+          <li class="links" id="links-p"><a href="#PokeCard">Pokémons</a></li>
+          <li class="links" id="links-a"><a href="#MyTeam">My Team</a></li>
+          <li class="links" id="links-c"><a href="#Competitions">Competitions</a></li></ul>
+    </nav>
+  </header>
+
+  <main>
+    <div id="cards-count"></div>
+    <section id="pokemon-list">
+    </section>
+  </main>
+  <footer>
+    Created by M1rn4 and Swarzstein under MIT licence
+  </footer>
```

Comment:
Consider making your footer similar to the one in the wireframe. This is one of the requirements of the project. Consider giving a top border to your footer, and change the font size and color of the text so it becomes more visible on the choosen background.⬇️
![image](https://user-images.githubusercontent.com/33268927/211139281-5019185e-8845-431b-a1c3-17fabc74f564.png)
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/211139295-fbba39ed-69e4-4047-be31-1d81008cc2b1.png) | ![image](https://user-images.githubusercontent.com/33268927/211139336-d765e972-9987-43fd-8ba4-2421432009f6.png) |